spec/tests/relevancy.fmf

```diff
@@ -14,12 +14,16 @@ description: >
     Environment is defined by one or more environment dimensions
     such as product, distro, collection, variant, arch, component.
     Relevancy consists of a set of rules of the form ``condition:
-    decision``. Should be a ``list``.
+    decision``. Should be a ``list`` of rules or a ``string`` with
```

Comment:
should we say out loud that the list items are "OR"-ed ?

Reply:
I think the this is not the right place to describe the whole concept in detail. But we could/should point to detailed test case relevancy docs. Probably add a link pointing [here](https://docs.fedoraproject.org/en-US/ci/test-case-relevancy/)?

Reply:
Added "The first matching rule wins." and a link to the detailed docs. @thrix, better now?